kiwi/cli.py

```diff
@@ -22,13 +22,15 @@
                [--target-arch=<name>]
                [--type=<build_type>]
                [--logfile=<filename>]
+               [--logsocket=<socketfile>]
```

Comment:
so AFAIU `--logsocket` and `--logfile` are not mutually exclusive and can be used together,
what would happen if Kiwi is run with both option pointing at the same path ? Like
`kiwi-ng --logsocket /tmp/log_socket --logfile /tmp/log_socket`

Reply:
> so AFAIU `--logsocket` and `--logfile` are not mutually exclusive and can be used together, what would happen if Kiwi is run with both option pointing at the same path ? Like `kiwi-ng --logsocket /tmp/log_socket --logfile /tmp/log_socket`
> 
> I understand that case should not be an issue, right ?

yes not an issue. You can use them together. As I wrote in the initial comment you need a listener for the specified logsocket. The python logging facility connects to the specified Unix Doman Socket. If there is no listener (aka server) that connect will fail and kiwi stops with an error message produced by that connection attempt. In the initial commit I posted a very simple example of such a listener such that people could try it out. You could also use ```--logsocket /dev/log``` which usually exists on Linux and points to syslog. In this case the kiwi logs appears on the syslog interface